rotkehlchen/exchanges/bitfinex.py

```diff
@@ -802,6 +795,60 @@ def _process_unsuccessful_response(
         raise AssertionError(f'Unexpected {self.name} response_case: {case}')
 
     def first_connection(self) -> None:
+        """Request:
```

Comment:
Shouldn't you also add a non-mock function that tests that the first connection actually workd properly and generates the pair tickers map? Since all these endpoints are public they should not require an API key so they should also be testable.

Reply:
Added!